<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="qLineEditIpAddr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ex:192.168.199.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSendData">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>521</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connecter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDisplatData">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>521</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelConnect">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connecter</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAddr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adresse serveur</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQTSocket">
    <property name="title">
     <string>QTSocket</string>
    </property>
   </widget>
   <addaction name="menuQTSocket"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
